### DISCUSSION TITLE
Being able to use emotes

### DISCUSSION THREAD

**Discussion by MateoskiI** [stoatchat/discussions#258] [Feature Suggestions]
🤔 When I was creating my account it wouldn't let me put an emoji in my username and neither in my bot name, I don't know if it's a bug or something they thought was right but hopefully they can fix it. 😃

**Comment by PeachiPrism** [comment]
I don't think emojis should be allowed in usernames. There is no reason to do that and it only causes problems for people looking to ping or add you.

The same goes for weird fonts that you cant type in usernames like "╲⎝⧹𝕯𝖆𝖗𝖐 𝕬𝖓𝖌𝖊𝖑⧸⎠╱", those should not be allowed as usernames. But you also don't want to blanket ban non English characters because that would limit people being able to have usernames in their native language.
